nautobot/extras/tests/test_api.py

```diff
@@ -559,37 +561,80 @@ def test_run_job_object_var(self):
         data = {
             "data": job_data,
             "commit": True,
+            "schedule": {
+                "name": "test",
+                "interval": "future",
+                "start_time": str(datetime.now() + timedelta(minutes=1)),
+            },
         }
 
         url = reverse("extras-api:job-run", kwargs={"class_path": "local/test_api/TestJob"})
         response = self.client.post(url, data, format="json", **self.header)
         self.assertHttpStatus(response, status.HTTP_200_OK)
 
-        url = reverse("extras-api:jobresult-detail", kwargs={"pk": response.data["result"]["id"]})
-        response = self.client.get(url, format="json", **self.header)
-        self.assertHttpStatus(response, status.HTTP_200_OK)
-        self.assertEqual(response.data["data"]["vars"]["var4"], d.pk)
+        job = ScheduledJob.objects.last()
+        self.assertEqual(job.kwargs["data"]["var4"], str(d.pk))
 
-    @skipIf(not rq_worker_running, "RQ worker not running")
     @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"], JOBS_ROOT=THIS_DIRECTORY)
     def test_run_job_object_var_lookup(self):
```

Comment:
Similar question on this test case as well.

Reply:
This one shouldn’t incur any lost coverage, since it takes the same code paths as the test above, minus the special logic in deserialization.